### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by saifudinbadani**
Screen Shot 2022-02-17 at 10 21 33 am

### CODE REVIEW

**Review by Mohasin-Haque — APPROVED** [review]
Code looks good to me.. You can go ahead

**Review by poornima-alugubelly — APPROVED** [review]
LGTM

File: common.css [hunk 1]
```diff
@@ -3,6 +3,7 @@
 @import "./css/login-signup.css";
 @import "./css/wishlist.css";
 @import "./css/cart.css";
+@import "./css/product-listing.css";
 
 :root{
     --bg-clr: #e5e5e5;
```
Comment (devsurajrai) [inline common.css:9]: Would suggest to use descriptive variable names.

**Review by devsurajrai — APPROVED** [review]
Everything looks good. Try to have the comments wherever possible.

**Pull request merged**